### ISSUE TITLE
can-types???!?

### ISSUE THREAD

**Issue by frank-dspeed** [canjs/canjs#3911]
I get this error and i am not sure why i maybe some one can make a suggestion

```
(node:4083) UnhandledPromiseRejectionWarning: Error: Error loading "can-types" at file:/media/sf_peep-server/applications/can-types.js
Error loading "can-types" from "applications@4.0.1#interfaces/checkin/checkin" at file:/media/sf_peep-server/applications/src/interfaces/checkin/checkin.js
Could not load 'can-types'
Is this an npm module not saved in your package.json?
    at file:/media/sf_peep-server/applications/node_modules/steal/ext/npm-extension.js:366:11
    at <anonymous>
(node:4083) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:4083) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

checkin.js
```javascript
import Component from 'can-component';
import DefineMap from 'can-define/map/';
import './styles.less';
import view from './checkin.stache';
import route from 'can-route';
export const ViewModel = DefineMap.extend(
  'Checkin',
  { seal:false },
  {
    channel: {
      default() {
        return route.data.channel;
      }
    }
  }
);

export default Component.extend({
  tag: 'interfaces-checkin',
  ViewModel,
  view
});

```

**Comment by frank-dspeed** [comment]
Adding can-types fixes the issue npm install --save can-types

i am Running CanJS4 latest maybe its needed by can-define or can-component and missing in package.json

**Comment by matthewp** [comment]
There was a package that was missing can-types that I updated this morning. Could you have an old lockfile?

**Comment by matthewp** [comment]
It was can-stache-bindings, fixed in 4.0.4. See if you have that version.

**Comment by frank-dspeed** [comment]
@matthewp ok your right thanks for info
